docs/index.md

```diff
@@ -22,9 +22,13 @@ provider "grafana" {
 <!-- schema generated by tfplugindocs -->
 ## Schema
 
-### Optional
+### Required
 
 - **auth** (String, Sensitive) API token or basic auth username:password. May alternatively be set via the `GRAFANA_AUTH` environment variable.
+- **url** (String) The root URL of a Grafana server. May alternatively be set via the `GRAFANA_URL` environment variable.
```

Comment:
Turns out, if you have the corresponding env variable exported in your console, docs generation will consider those params as optional